packages/@aws-cdk/aws-events/test/test.input.ts

```diff
@@ -102,6 +102,49 @@ export = {
       }));
       test.done();
     },
+
+    'can use token from cfnParameter'(test: Test) {
```

Comment:
This feels like a duplicate of the test in `test.rule.ts`. Does this check anything additional that the other test case doesn't?

Reply:
Let me check and remove if it is duplicate.